src/vulkan_state.cpp

```diff
@@ -62,6 +62,17 @@ static void fillQueueInfo(VkDeviceQueueCreateInfo &info, uint32_t idx,
     info.pQueuePriorities = priorities.data();
 }
 
+VkFormatProperties2 getPhysicalDeviceProperties(const InstanceState &inst,
+                                                VkPhysicalDevice phy,
+                                                auto &fmt) {
```

Comment:
Change auto &fmt to VkFormat fmt (or whatever the concrete type is). No need for reference because it is just an enum (aka integer).

Reply:
didn't know this was a type but it makes sense